Chapter3/README.md

```diff
@@ -0,0 +1,202 @@
+---
+page_type: sample
+author: derisen
+languages:
+- javascript
+- typescript
+- csharp
+products:
+- angular
+- dotnet
+- azure-active-directory
+- microsoft-graph-api
+- azure-app-services
+- azure-storage
+description: "Deploying a Multi-tenant (SaaS) Angular Single-page Application (SPA) that Authenticates users with Azure AD and calls a protected ASP.NET Core web API"
+urlFragment: "ms-identity-javascript-angular-spa-aspnet-webapi-multitenant/Chapter3"
+---
+
+# Deploying a Multi-tenant (SaaS) Angular Single-page Application (SPA) that Authenticates users with Azure AD and calls a protected ASP.NET Core web API
+
+In this guide, we will deploy our project components, **TodoListAPI** and **TodoListSPA**, to Azure Cloud. For **TodoListAPI**, we will use **Azure App Services**, while for **TodoListSPA**, we will generate a static website and upload it to **Azure Storage**.
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/download/) must be installed to run this sample.
+- [Dotnet Core SDK](https://dotnet.microsoft.com/download) must be installed to run this sample.
+- *At least* **two** Azure Active Directory (Azure AD) tenants if you want to test deployment end-to-end. For more information on how to get an Azure AD tenant, see [How to get an Azure AD tenant](https://azure.microsoft.com/documentation/articles/active-directory-howto-tenant/).
+- On each tenant, *at least* **one** admin account and **one** non-admin/user account for testing purposes.
+- An **Azure subscription**. This sample uses **Azure App Services** and **Azure Storage**.
+- A modern Browser. This sample uses **ES6** conventions and will not run on **Internet Explorer**.
+- We recommend [VS Code](https://code.visualstudio.com/download) for running and debugging this cross-platform application.
+- We recommend [Azure Tools for VS Code Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack) for interacting with Azure services directly within VS Code.
+
+## Setup
+
+Using a command line interface such as VS Code integrated terminal, follow the steps below:
+
```

Comment:
Since we've removed the code from this directory, we should tell the user that they'll have to operate from the Chapter2 directory OR tell them to `cp -r ../Chapter2/TodoListSPA/ .; cp -r ../TodoListAPI/ .`

Reply:
(and the equivalent Windows command or powershell command if they're not operating from bash or similar prompt? I don't know how common this use case might be)

Reply:
Done -added a note (both Powershell and VS code understands bash commands, so that's usually a non-issue)